tutorials/wordle.md

```diff
@@ -169,7 +169,7 @@ Rollkit on our codebase.
 To install the Rollkit app to Ignite, run the following command:
 
 ```bash
-ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.0
+ignite app install github.com/ignite/apps/rollkit@rollkit/v0.2.1
```

Comment:
IIUC @yarikbratashchuk - this could use the variable instead, right?

Reply:
yes, i'll make a pr with it and will set it to v0.2.1